### ISSUE TITLE
Missing support double among other errors

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#433]
0F2F10E6EFF79C58C5A201FB1D5C
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/cffb6800-0420-4be9-9c70-ccd0183b8cfa)

Playing support double North should double 1S to show 3-card support.

Without the double 3D is NOT forcing, and the correct bid is 2H responders reverse and GF

With the double the correct bid in south is still 2H.

North would then respond 2N/3N

As the double is not clearly defined this could happen
https://shorturl.at/rRVZ1

Even after the NF (Invitational) 3D North mess up, trying to show support instead of bidding the natural 3N. Even if it was forcing 3N would still be the right bid.

**Comment by EdwardPiwowar** [comment]
No support double here - responder must show a major.
Next BBA
![image](https://github.com/EdwardPiwowar/BBA/assets/150140020/9f33b437-e177-47a1-a7da-7bfa1564c228)

**Comment by ThorvaldAagaard** [comment]
D from North must show 3D, when playing support doubles (Defined by GIB). Without support doubles it can show 4H. 
After overcall 1H, North can by 1S to show 4, and double is support double if on, or else it is a hand with extra, but without 4S

**Comment by EdwardPiwowar** [comment]
Everything is clear, but in BBA I don't have double support for diamonds.

**Comment by ThorvaldAagaard** [comment]
Then please add.

According to Larry Cohen not using it after 1D response is an exception
https://www.larryco.com/bridge-articles/support-doubles

![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/2b58f1b0-d67f-4a91-b67e-31a38d39a60c)

**Comment by EdwardPiwowar** [comment]
Later, you have to split the convention into two and change a lot in the implementation.

**Comment by ThorvaldAagaard** [comment]
Yes, I understand, but you have to get the defaults right before adding exceptions. Here you added the exception as default (or did not implement the full convention).

I would be nice if we could invent a way to add exceptions for any conventions, but that is not simple.
